client/app/components/visualizations/VisualizationRenderer.jsx

```diff
@@ -42,12 +43,17 @@ export default function VisualizationRenderer(props) {
     setFilters(combineFilters(filtersRef.current, props.filters));
   }, [props.filters]);
 
+  const cleanColumnNames = useMemo(
+    () => map(data.columns, col => ({ ...col, name: getColumnCleanName(col.friendly_name) })),
```

Comment:
@kravets-levko it seemed to make sense to move `getColumnCleanName` out of Table vis and rather apply it to all visualizations, do you see any problems may happen with this change? (that method is used to remove `::filter` from column names for example)